modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx

```diff
@@ -120,7 +127,7 @@ export function RightSidebarObjectDefinitionDetails({
 					newNonRelationshipObjectFieldsInfo
 				);
 				setValues(selectedObjectDefinition);
-				setWorkflowLabel(objectDefinitionInfo.workflowDefinitionTitle);
+				setWorkflowInfo(objectDefinitionInfo);
```

Comment:
I find odd that this is named `objectDefinitionInfo`. I noticed we have broadened the scope of `getDbTableName` and had to rename it so it is more generic, but still...

Reply:
I am sure the back-end has its reasons, but this name makes me ask myself: if it is "object definition info", why it is not included in the standard `getObjectDefinitionByExternalReferenceCode` endpoint response?

Reply:
i think that were some limitations about this, i'm not sure about it now...but i also agree it would be more clear for it to come inside the object definition itself